specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/ManagedEnvironmentsMaintenanceConfigurations.json

```diff
@@ -279,7 +279,8 @@
       },
       "required": [
         "weekDay",
-        "startHourUtc"
+        "startHourUtc",
+        "durationHours"
```

Comment:
sorry one more question. 

Existing users of this API are now expected to send this field ? 

Will that not break the customers who are not sending this today ?

Reply:
This API is not public. It is being introduced in current api-version release branch.
Refer https://github.com/Azure/azure-rest-api-specs/pull/30769